### PR TITLE
Fedora 29 build.

### PR DESCRIPTION
**Pull request by Tjzabel**
This updates the RPM signing key to be used with all F29 packages.

### CODE REVIEW

File: tigeros-repos-1.0/tigeros.repo [hunk 1]
```diff
@@ -17,4 +17,3 @@ metadata_expire=7d
 gpgcheck=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-TigerOS-$releasever-primary
```
Comment (ct-martin) [inline tigeros-repos-1.0/tigeros.repo]: Does this mean that the files need to be named RPM-GPG-KEY-TigerOS-3.0-primary?

Comment (Tjzabel) [reply tigeros-repos-1.0/tigeros.repo]: @ct-martin nope. *Usually* it would, but the `$releasever` variable refers to the Fedora version in this case.

Comment (Tjzabel) [reply tigeros-repos-1.0/tigeros.repo]: Actually, I will look more into this tonight.

It might need to be `RPM-GPG-Key-TigerOS-$vers-primary`

**Comment by Tjzabel** [comment]
@ct-martin In general, I'm not even sure why there is a sym link to another GPG key. I am going to remove the sym link and update the GPG key name.

I *believe* the only thing that needs the GPG name is the tigeros.repo.

**Pull request merged**